src/read.c

```diff
@@ -2809,8 +2840,23 @@ static avifResult avifParseItemPropertyAssociation(avifMeta * meta, const uint8_
                 avifProperty * dstProp = (avifProperty *)avifArrayPush(&item->properties);
                 AVIF_CHECKERR(dstProp != NULL, AVIF_RESULT_OUT_OF_MEMORY);
                 *dstProp = *srcProp;
+
+                if (isTransformative) {
+                    transformativePropertySeen = AVIF_TRUE;
+                }
```

Comment:
Optional: I think it is clearer to move this if statement (lines 2844-2846) to the end of the inner for loop's body, between line 2873 and line 2874. This makes it clear that this if statement is executed in every iteration of the inner for loop.